nativescript-angular/router/ns-router-link.ts

```diff
@@ -30,30 +30,41 @@ import {routerLog} from "../trace";
  */
 @Directive({ selector: '[nsRouterLink]' })
 export class NSRouterLink {
-  private commands: any[] = [];
-  @Input() target: string;
-  @Input() queryParams: { [k: string]: any };
-  @Input() fragment: string;
+    private commands: any[] = [];
+    @Input() target: string;
+    @Input() queryParams: { [k: string]: any };
+    @Input() fragment: string;
 
-  /**
-   * @internal
-   */
-  constructor(private router: Router, private route: ActivatedRoute) { }
+    urlTree: UrlTree;
```

Comment:
I think is unnecessary `urlTree`. In the [stock RouterLink](https://github.com/angular/angular/blob/master/modules/%40angular/router/src/directives/router_link.ts#L155-L157) it is used only to update `href` of a tags.